modules/core/src/main/scala/com.snowplowanalytics.iglu/client/resolver/registries/Utils.scala

```diff
@@ -72,6 +76,40 @@ private[registries] object Utils {
         repoFailure(e).asLeft
     }
 
+  def unsafeEmbeddedList(path: String, modelMatch: Int): Either[RegistryError, SchemaList] =
+    try {
+      val d = new File(getClass.getResource(path).getPath)
+      val schemaFileRegex: Regex = (".*?" + // path to file
+        "([a-zA-Z0-9-_.]+)/" +            // Vendor
+        "([a-zA-Z0-9-_]+)/" +             // Name
+        "([a-zA-Z0-9-_]+)/" +             // Format
+        "([1-9][0-9]*)-(\\d+)-(\\d+)$").r // MODEL, REVISION and ADDITION
+      if (d.exists && d.isDirectory) {
+        d.listFiles
+          .filter(_.isFile)
+          .toList
+          .filter(_.getName.startsWith(modelMatch.toString))
```

Comment:
Good catch! It makes it a bit awkward to put these because there isn't a neutral `SchemaKey` element. It would have to be inside anther filter. `100-` could be avoided with `s"${modelMatch.toString}-}`, see my new commit.